docs/v2 (system-addon)/data_dictionary.md

```diff
@@ -1,87 +1,82 @@
 # Activity Stream Pings
```

Comment:
@emtwo Can you review 54bef6d4d5b78ac90e272d69f90d54d1ef789e6b to make sure everything looks reasonable?